.github/workflows/release.yml

```diff
@@ -180,6 +107,26 @@ jobs:
         run: |
           # List out the docker images on the system
           docker images
+      - name: Download installer
+        uses: actions/download-artifact@v3.0.1
+        with:
+          name: installer
+          path: assets/*
+      - name: Download NuGet packages
+        uses: actions/download-artifact@v3.0.1
+        with:
+          name: nugets
+          path: nugets/*
+      - name: Download zips
+        uses: actions/download-artifact@v3.0.1
+        with:
+          name: zips
+          path: zip/*
+      - name: Deploy
+        uses: Particular/push-octopus-package-action@v1.0.0
+        with:
+          octopus-deploy-api-key: ${{ secrets.OCTOPUS_DEPLOY_API_KEY }}
+          additional-metadata-paths: ServiceControlMetadata.json
```

Comment:
Don't you need to download this one since it was created by the previous job?

Reply:
No, because the file is now created in the `docker` job (previously, it was in the `release` job) at line 91, right after building the docker images. Being in `docker` job is already where it's need to be at the right time.